test/Interop/Cxx/class/Inputs/module.modulemap

```diff
@@ -2,6 +2,10 @@ module AccessSpecifiers {
   header "access-specifiers.h"
 }
 
+module Initializer {
```

Comment:
`MemberwiseInitializer`? Similarly in all other names and file names.

Reply:
Done